docs/code/storage_config.py

```diff
@@ -12,8 +12,8 @@
 """Get cloud client"""
```

Comment:
`docs/source/advanced_features/storage_config.rst` the config name and dataset name should be modified.